docs/commands.md

```diff
@@ -471,7 +471,8 @@ _Arguments_
 * _ID_: [Session](glossary.html#session) unique ID (if not given, RedisGears will generate one)
 * _DESCRIPTION_: Optional [Session](glossary.html#session) description
 * _UPGRADE_: If the session with this name already exists, replace it.
-* _REPLACE_WITH_: Set the new [Session]() as a replacement of the session give by this argument.
+* _REPLACE_WITH_: Set the new [Session](glossary.html#session) as a replacement of the session give by this argument.
```

Comment:
```
* _REPLACE_WITH_: Replace the session with a this, as a new [Session](glossary.html#session).
```

Reply:
I did not touch this sentence and I think it was reviewed by @rrelledge

Reply:
Either way it's poor english. Worst case change "give" to "specified"